communications/newsletters/newsletter_12_Nov2019.md

```diff
@@ -56,7 +76,7 @@ Thanks to several of you for joining Kirstie Whitaker, Yo Yehudi, and Malvika Sh
 
 ### Features and Shoutouts
```

Comment:
Do you want to shout out to Binder for getting CZI money?

Reply:
Where does that fit? Maybe where we are mentioning binderhub?

Reply:
Added in this section (acknowledgements and celebrations) - does that make sense? Or are these supposed to be just TW related?

Reply:
I added stuff related to TW, but I suppose it is nice to shoutout to the collaborators and contributors for their successes 😃